thesis-gui/src/types/quizzes-list.ts

```diff
@@ -0,0 +1,7 @@
+export type Quiz = {
+    questionsSetName: string,
+    questionsSetKey: string,
+    numberOfQuestionsInSet: number
+}
+
+export type QuizzesList = Quiz[];
```

Comment:
Are you sure this type alias is needed? IMO it's better just to use `Quiz[]` - you can see that the type is a simple array.